app/src/main/java/io/homeassistant/companion/android/settings/qs/views/ManageTilesView.kt

```diff
@@ -0,0 +1,144 @@
+import androidx.core.graphics.drawable.toBitmap
+import androidx.fragment.app.FragmentManager
+import com.maltaisn.icondialog.IconDialog
+import com.maltaisn.icondialog.pack.IconPack
+import com.maltaisn.icondialog.pack.IconPackLoader
+import com.maltaisn.iconpack.mdi.createMaterialDesignIconPack
+import io.homeassistant.companion.android.common.R
+import io.homeassistant.companion.android.database.AppDatabase
+import io.homeassistant.companion.android.database.qs.TileEntity
+import io.homeassistant.companion.android.settings.qs.ManageTilesViewModel
+
+private lateinit var iconPack: IconPack
+
+@Composable
+fun ManageTilesView(
+    viewModel: ManageTilesViewModel,
+    iconDialog: IconDialog,
+    childFragment: FragmentManager
+) {
+    val context = LocalContext.current
+    var expandedTile by remember { mutableStateOf(false) }
+    var expandedEntity by remember { mutableStateOf(false) }
+    Column(modifier = Modifier.padding(20.dp)) {
+        TextButton(onClick = { expandedTile = true }) {
+            Text(
+                text = stringResource(R.string.tile_select) + viewModel.selectedTile.value,
+                fontSize = 15.sp
+            )
+        }
+        DropdownMenu(expanded = expandedTile, onDismissRequest = { expandedTile = false }) {
+            val tileNameArray = stringArrayResource(id = io.homeassistant.companion.android.R.array.tile_name)
+            val tileIdArray = stringArrayResource(id = io.homeassistant.companion.android.R.array.tile_ids)
+            for ((tileName, tileId) in tileNameArray.zip(tileIdArray)) {
+                DropdownMenuItem(onClick = {
+                    viewModel.selectedTile.value = tileId
+                    expandedTile = false
+                    if (viewModel.currentTile() != null) {
+                        viewModel.tileLabel.value = viewModel.currentTile()!!.label
+                        viewModel.tileSubtitle.value = viewModel.currentTile()!!.subtitle
+                        viewModel.selectedEntityId.value = viewModel.currentTile()!!.entityId
+                        viewModel.selectedIcon.value = viewModel.currentTile()!!.iconId
+                        val loader = IconPackLoader(context)
+                        iconPack = createMaterialDesignIconPack(loader)
+                        iconPack.loadDrawables(loader.drawableLoader)
+                        val iconDrawable = viewModel.selectedIcon.value?.let { iconPack.getIcon(it)?.drawable }
+                        if (iconDrawable != null) {
+                            val icon = DrawableCompat.wrap(iconDrawable)
+                            icon.setColorFilter(context.resources.getColor(io.homeassistant.companion.android.R.color.colorAccent), PorterDuff.Mode.SRC_IN)
+                            viewModel.icon.value = icon
+                        }
+                    }
+                }) {
+                    Text(tileName)
+                }
+            }
+        }
+        Text(

```

Comment:
It might be nice to add a `Divider` here to visually separate the tile selection (general) from the individual settings (specific)